components/lcd/esp_lcd_lt8912b/esp_lcd_lt8912b.c

```diff
@@ -0,0 +1,718 @@
+/*
+ * SPDX-FileCopyrightText: 2023-2024 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <string.h>
+#include <stdlib.h>
+#include <sys/cdefs.h>
+#include "freertos/FreeRTOS.h"
+#include "freertos/task.h"
+#include "esp_lcd_panel_interface.h"
+#include "esp_lcd_panel_io.h"
+#include "esp_lcd_panel_vendor.h"
+#include "esp_lcd_panel_ops.h"
+#include "esp_lcd_panel_commands.h"
+#include "driver/gpio.h"
+#include "esp_log.h"
+#include "esp_check.h"
+#include "esp_lcd_lt8912b.h"
+
+#define ENABLE_TEST_PATTERN 0
+
+#define lt8912b_CMD_OPCODE (0x22)
```

Comment:
Nitpick Style: can we have all macros in capital letters?
```suggestion
#define LT8912B_CMD_OPCODE (0x22)
```